src/VatRates/TaxesEuropeDatabaseClient.php

```diff
@@ -0,0 +1,93 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\Vat\VatRates;
+
+use DateTime;
+use SoapClient;
+use SoapFault;
+
+final class TaxesEuropeDatabaseClient implements ResolvesVatRates
+{
+    const WSDL = 'https://ec.europa.eu/taxation_customs/tedb/ws/VatRetrievalService.wsdl';
+    const SOAP_CONFIG = [
+        'version' => SOAP_1_1,
+        'cache_wsdl' => WSDL_CACHE_MEMORY,
+    ];
```

Comment:
Both of these are default so we can omit them.

Reply:
Will do